src/test/java/co/rsk/federate/signing/hsm/message/PowHSMSignerMessageBuilderTest.java

```diff
@@ -526,9 +430,15 @@ private BtcTransaction createPegout(List<Coin> outpointValues, Address destinati
         return pegoutBtcTx;
     }
 
-    private void addWitness(BtcTransaction pegoutBtcTx, int inputIndex) {
-        TransactionWitness txWitness = new TransactionWitness(1);
-        txWitness.setPush(0, new byte[]{0x1});
-        pegoutBtcTx.setWitness(inputIndex, txWitness);
+    private void addCommonPegoutLogs(List<LogInfo> logs, BtcTransaction pegoutBtcTx) {
```

Comment:
Shouldn't all transactions have batch_pegout_created event?

Reply:
It shouldn't when the pegout is a migration.